frontend-react/src/__mocks__/LookupTableMockServer.ts

```diff
@@ -23,12 +27,14 @@ const lookupTables: LookupTable[] = [1, 2, 3].map((i) => ({
     tableSha256Checksum: "checksum",
 })) as LookupTable[];
 
-const lookupTableData: ValueSet[] = [1, 2, 3].map((i) => ({
+const lookupTableData: ApiValueSet[] = [1, 2, 3].map((_i) => ({
     name: "sender_automation_value_set",
-    createdBy: `test${i}@example.com`,
-    createdAt: "now",
+    created_by: "",
```

Comment:
this is an inconsequential change, but better mirrors the data that is actually coming back from the API